src/app/shared/shared.module.ts

```diff
@@ -37,6 +42,7 @@ export const PIPES_AND_COMPONENTS = [
   declarations: PIPES_AND_COMPONENTS,
   exports: PIPES_AND_COMPONENTS,
   entryComponents: [AdrReasonModalComponent, PreventLeavePageModalComponent],
-  providers: [FilterRecordPipe, PendingChangesService],
+  providers: [FilterRecordPipe, PendingChangesService, TechRecordHelpersService],
```

Comment:
Dont think you need this. TechRecordHelpersService already `@Injectable({
  providedIn: 'root'
})` that should be enough